External/FEXCore/Source/Interface/Core/ArchHelpers/CodeEmitter/Buffer.h

```diff
@@ -87,6 +87,11 @@ namespace FEXCore::ARMEmitter {
         return Size;
       }
 
+      template<typename T>
+      size_t GetCursorOffsetFromAddress(T* Address) {
```

Comment:
```suggestion
      size_t GetCursorOffsetFromAddress(const T* Address) const {
```

would make this a little more flexible for use

Reply:
Added const-ness